src/actions/posts.js

```diff
@@ -790,7 +796,7 @@ export function getProfilesAndStatusesForPosts(postsArrayOrMap, dispatch, getSta
     return Promise.all(promises);
 }
 
-export function getNeededAtMentionedUsernames(state, posts) {
+export function getNeededAtMentionedUsernames(state: GlobalState, posts: Array<Post>): ?Set<string> {
```

Comment:
Can this just return a `Set<string>` without the `?`? It doesn't look like we ever return null here